x/ccv/consumer/keeper/keeper_test.go

```diff
@@ -90,60 +90,54 @@ func TestPacketMaturityTime(t *testing.T) {
 	defer ctrl.Finish()
 
 	now := time.Now().UTC()
-	nsNow := uint64(now.UnixNano())
 	packets := []types.MaturingVSCPacket{
 		{
 			VscId:        2,
-			MaturityTime: nsNow,
+			MaturityTime: now,
 		},
 		{
 			VscId:        1,
-			MaturityTime: nsNow - 15,
+			MaturityTime: now.Add(-time.Hour),
 		},
 		{
 			VscId:        5,
-			MaturityTime: nsNow - 30,
+			MaturityTime: now.Add(-2 * time.Hour),
 		},
 		{
 			VscId:        6,
-			MaturityTime: nsNow + 10,
+			MaturityTime: now.Add(time.Hour),
 		},
 	}
 	expectedGetAllOrder := packets
-	// sorting by VscId
+	// sorting by MaturityTime
 	sort.Slice(expectedGetAllOrder, func(i, j int) bool {
-		return expectedGetAllOrder[i].VscId < expectedGetAllOrder[j].VscId
+		return expectedGetAllOrder[i].MaturityTime.Before(expectedGetAllOrder[j].MaturityTime)
 	})
 	expectedGetElapsedOrder := []types.MaturingVSCPacket{}
 	for _, packet := range packets {
-		// only packets with MaturityTime <= nsNow
-		if packet.MaturityTime <= nsNow {
+		// only packets with MaturityTime <= now
+		if !packet.MaturityTime.After(now) {
 			expectedGetElapsedOrder = append(expectedGetElapsedOrder, packet)
 		}
 	}
-	// sorting by VscId
+	// sorting by MaturityTime
 	sort.Slice(expectedGetElapsedOrder, func(i, j int) bool {
-		return expectedGetElapsedOrder[i].VscId < expectedGetElapsedOrder[j].VscId
+		return expectedGetAllOrder[i].MaturityTime.Before(expectedGetAllOrder[j].MaturityTime)
 	})
 
 	for _, packet := range packets {
 		ck.SetPacketMaturityTime(ctx, packet.VscId, packet.MaturityTime)
 	}
 
-	for _, packet := range packets {
```

Comment:
Should we keep this check with the new "exists" method?

Reply:
The exists method is only used in testing. Do we need UTs for those?

Reply:
Added it anyway.